appengine/storage.js

```diff
@@ -169,10 +169,10 @@ BlocklyStorage.monitorChanges_ = function(workspace) {
     var xmlText = Blockly.Xml.domToText(xmlDom);
     if (startXmlText != xmlText) {
       window.location.hash = '';
-      workspace.removeChangeListener(bindData);
+      workspace.removeChangeListener(change);
     }
   }
-  var bindData = workspace.addChangeListener(change);
+  workspace.addChangeListener(change);
```

Comment:
While true in practice, the `addChangeListener(..)` documentation does not guarantee the returned function is the same as the passed function.

Reply:
Nowhere in Blockly are we still using the returned function (this was the last straggler).  I've changed the docs to indicate that the return value is obsolete and to be ignored.